contracts/treasury/interfaces/ICollector.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.10;
+
+import {IERC20} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+
+interface ICollector {
+  /**
```

Comment:
Would be nice the interface functions follow the same order as the functions in the implementation (think it increases readability and maintainabilty)